packages/bento-design-system/src/TextArea/createTextArea.tsx

```diff
@@ -0,0 +1,65 @@
+import { useTextField } from "@react-aria/textfield";
+import { useRef } from "react";
+import { Box } from "../internal";
+import { LocalizedString } from "../util/LocalizedString";
+import { inputRecipe } from "../Field/Field.css";
+import { FieldProps } from "../Field/FieldProps";
+import { bodyRecipe } from "../Typography/Body/Body.css";
+import { FieldType } from "../Field/createField";
+import { InputConfig } from "../Field/Config";
+
+type Props = FieldProps<string> & {
+  placeholder: LocalizedString;
+  isReadOnly?: boolean;
+};
```

Comment:
can you also include the standard `rows` attribute to pass down to the textarea, so that a user can control the number of rows?